helm/charts/hpe-csi-driver/values.schema.json

```diff
@@ -163,7 +163,8 @@
             "default": 60,
             "minimum": 60,
             "maximum": 360
-        }
+        },
+	"global": {}
```

Comment:
Replace the tab with spaces.

Reply:
Daym, how can this have happened? Fixed.